scenes/game/life_scene.gd

```diff
@@ -23,7 +24,7 @@ func set_clock(seconds : int):
 	var minutes = seconds / 60.0
 	var seconds_display = fmod(seconds, 60)
 	var mmss_string : String = "%s%02d:%02d" % [negative_str, minutes, seconds_display]
-	clock.text = mmss_string
+	$Clock.text = mmss_string
```

Comment:
can you actually go back to using the variable "clock" that is from the top of this file.
And actually if you can replace the other $Clock with clock that would be great.
My thinking here is that it is much more annoying to repeat the $ style stuff throughout a file and I'd rather have them defined to variables at the top.

Reply:
Alternatively, you could just revert this file since you didn't actually change anything?

Reply:
I'm rather puzzled since I don't remember making this change at all and I don't know why I would, but I suppose I must have. changed it back to the clock variable